backend/services/EventHandler.py

```
@@ -30,23 +30,17 @@ def parse_event_datetime(
-    if not image:
-        return None
-    
-    filename = generate_secure_filename(user_id, ".jpg")
-    save_upload_file(image, upload_dir, filename)
-    return filename
+
 
 async def handle_event_upload(
     media_file: Optional[UploadFile],
     folder_name: str
 ) -> Dict[str, str]:
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)